komodo-core/src/main/resources/org/komodo/repository/initialContent.xml

```diff
@@ -6,6 +6,7 @@
 		<tko:library jcr:primaryType="tko:library" />
 		<tko:environment jcr:primaryType="tko:environment">
             <tko:validation jcr:primaryType="nt:unstructured" />
+            <tko:servers jcr:primaryType="nt:unstructured" />
```

Comment:
I'm thinking that on startup of VDBBuilder for the first time, we may be able to detect that default server does not exist - then use that to somehow prompt user to enter the properties...